docs/source/api/python/mantid/geometry/ComponentInfo.rst

```diff
@@ -0,0 +1,26 @@
+===============
+ ComponentInfo
+===============
+
+This is a python binding to the C++ class Mantid::Geometry::ComponentInfo.
+
+--------
+Purpose
+--------
+The purpose of the ComponentInfo object is to allow the user to access information about the components which are part of an instrument. The ComponentInfo object can be used to access information such as the number of components in the instrument, the absolute position of a component as well as the absolute rotation of a component. There are many other methods available as well.
```

Comment:
The purpose of the ComponentInfo object is to allow the user to access geometric information about the components which are part of beamline. A component is any physical item or group of items that is registered for the purpose of data reduction. The ComponentInfo object can be used to access information such as the total number of components in the instrument, the absolute position of a component as well as the absolute rotation of a component.

Reply:
I'll add in this change